lib/ExecutionControlEpic/DiagnosticsFeature/Actions/JumpTo.js

```diff
@@ -11,12 +11,17 @@ export function jumpTo(
   range: Range,
   pending: boolean,
 ): JumpToAction {
```

Comment:
It should be either a JumpToAction or void then, because jumping to a non-existent file is not really pertinent, is it ?
In case of null path nothing is shown in the diagnostic (no button for them to click)

Reply:
It is not indeed. But in case you don't want to return anything, you should update the return type to reflect that.